src/components/DuosHeader.jsx

```diff
@@ -0,0 +1,540 @@
+      { label: 'DACs', link: '/manage_dac' },
+      { label: 'Users', link: '/admin_manage_users' },
+      { label: 'Institutions', link: '/admin_manage_institutions' },
+      { label: 'Library Cards', link: '/admin_manage_lc' }
+    ],
+    isRendered: (user) => user.isAdmin
+  },
+  {
+    label: 'SO Console',
+    link: '/signing_official_console/researchers',
+    children: [
+      { label: 'Researchers', link: '/signing_official_console/researchers' },
+      { label: 'DAR Requests', link: '/signing_official_console/dar_requests' },
+      { label: 'Data Submitters', link: '/signing_official_console/data_submitters', isRendered: () => checkEnv(envGroups.NON_STAGING) },
+      { label: 'My Datasets', link: '/datalibrary/myinstitution' }
+    ],
+    isRendered: (user) => user.isSigningOfficial
+  },
+  {
+    label: 'DAC Chair Console',
+    link: '/chair_console',
+    search: 'chair_console',
+    children: [
+      { label: 'DAR Requests', link: '/chair_console' },
+      { label: 'DAC Members', link: '/manage_dac' },
+      { label: "My DAC's Datasets", link: '/dac_datasets' }
+    ],
+    isRendered: (user) => user.isChairPerson
+  },
+  {
+    label: 'DAC Member Console',
+    link: '/member_console',
+    search: 'member_console',
+    children: [
+      { label: 'DAR Requests', link: '/member_console' },
+      { label: 'Datasets', link: '/dataset_catalog' },
+    ],
+    isRendered: (user) => user.isMember
+  },
+  {
+    label: 'Researcher Console',
+    link: '/dataset_catalog',
+    search: 'dataset_catalog',
+    children: [
+      { label: 'Data Catalog', link: '/dataset_catalog' },
+      { label: 'Data Library', link: '/datalibrary', search: 'datalibrary' },
+      { label: 'DAR Requests', link: '/researcher_console' },
+      { label: 'Data Submissions', link: '/dataset_submissions', isRenderedForUser: (user) => user?.isDataSubmitter }
+    ],
+    isRendered: (user) => user.isResearcher && !isOnlySigningOfficial(user)
+  }
+];
+
+const NavigationTabsComponent = (props) => {
+  const {
+    orientation,
+    makeNotifications,
+    navbarDuosIcon, duosLogoImage, DuosLogo, navbarDuosText,
+    currentUser, signOut, isLogged,
+    contactUsButton, showRequestModal, supportrequestModal,
+    tabs, initialTab, initialSubTab,
+    onSubtabChange
+  } = props;
+  const [selectedMenuTab, setSelectedMenuTab] = useState(false);
+  const [selectedSubTab, setSelectedSubTab] = useState(false);
+
+  useEffect(() => {
+    setSelectedMenuTab(initialTab === -1 ? false : initialTab);
+    setSelectedSubTab(initialSubTab === -1 ? false : initialSubTab);
+  }, [initialTab, initialSubTab]);
+
+  return (
+    <div className={`navbar-logged ${orientation === 'vertical' ? 'navbar-vertical' : ''}`}>
+      {makeNotifications()}
+      <ul className="navbar-main">
+        <div style={{ width: '100%', display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
+          <Link
+            isRendered={orientation === 'horizontal'}
```

Comment:
`isRendered` is not a property on `Link`

Reply:
Oh no, @rushtong missed this. I'll be more careful next time. I never thought I'd miss Typescript so much.

Reply:
Hah, no worries @PintoGideon - TS conversion is also one of the things we're trying to do alongside all of the other migrations we're making. I don't necessarily recommend that for current JSX work, but it is something we want to move to.